bungin/SoundScaped
# Su hash fix

signup works with two caveats:

1: you can make duplicates with the same username and email.
2: the signup form does not clear on submit (should be an easy fix with html).

Please test prior to merge, i already have but a second set of eyes might see something new.
Testing instructions:VVVVVVVVVVVVVVV 
git checkout main(ensure you start in main for consistency), then
git pull (update main), then
git fetch -a (gets and updates(?) all branches), then
git checkout suHashFix (this branch includes what is already in main)
